src/Pyz/Yves/CatalogPage/Theme/default/templates/page-layout-catalog/page-layout-catalog.twig

```diff
@@ -2,7 +2,16 @@
 
 {% block contentClass %}page-layout-main page-layout-main--catalog-page{% endblock %}
 
+{% set catalogProductListId = 'catalog-product-list' %}
```

Comment:
We don't need to redefine it, as we have it in core. Please, recheck in all templates/components